docker-compose.yml

```diff
@@ -274,6 +274,31 @@ services:
         - /var/run/docker.sock:/var/run/docker.sock:ro
         - /lib/modules:/lib/modules:ro
 
+    acme-mailcow:
+      depends_on:
+        - nginx-mailcow
+      image: mailcow/acme
+      build: ./data/Dockerfiles/acme
+      # All domains to be included in the certificate
+      command: ${MAILCOW_HOSTNAME}
+      #autodiscover.${MAILCOW_HOSTNAME} autoconfig.${MAILCOW_HOSTNAME}
+      environment:
+        - CONTAINERS_RESTART=mailcowdockerized_postfix-mailcow_1 mailcowdockerized_dovecot-mailcow_1 mailcowdockerized_nginx-mailcow_1
+      volumes:
+        - ./data/web/.well-known/acme-challenge:/var/www/acme:rw
+        - ./data/assets/ssl:/var/lib/acme/:rw
+        - /var/run/docker.sock:/var/run/docker.sock:ro
+      dns:
+        - 172.22.1.254
+      dns_search: mailcow-network
+      # TODO: find some way to use always
+      # do not restart the container after exit. Things get worse when we hit let's encrypt's ratelimit.
+      restart: "no"
```

Comment:
There is also `ON_FAILURE:`

![image](https://user-images.githubusercontent.com/858296/26901043-af434ef6-4bd4-11e7-8f63-f8d11de2381e.png)

https://docs.docker.com/engine/reference/run/#restart-policies-restart